webpack/assets/javascripts/react_app/components/CPUCoresInput/__tests__/CPUCoresInput.test.js

```diff
@@ -5,13 +5,39 @@ const props = {
   label: 'CPUs',
 };
 
+const warningProps = {
+  label: 'CPUs',
+  value: 11,
+};
+
+const errorProps = {
+  label: 'CPUs',
+  value: 22,
+};
+
 const fixtures = {
   'should render with default props': {
     ...props,
   },
 };
 
+const warningFixtures = {
+  'should render with warning message': {
+    ...warningProps,
+  },
+};
+
+const errorFixtures = {
+  'should render with warning message': {
+    ...errorProps,
+  },
+};
```

Comment:
Can you merge this into 1 object? 
`fixtures ={ 'should..default...':{},  'should..warning...':{}, 'should..error...':{}}`